examples/experimental/mnist_ptl/ptl.py

```diff
@@ -0,0 +1,106 @@
+import torch
```

Comment:
blocking: let's not put the PTL support in experimental. There is actually nothing to experiment about.